mule-extensions-api/src/main/java/org/mule/runtime/extension/api/values/MuleValueAdapter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.values;
+
+import static java.util.Collections.unmodifiableSet;
+import static java.util.stream.Collectors.toCollection;
+
+import org.mule.runtime.api.value.Value;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class MuleValueAdapter implements Value {
+
+  private final org.mule.sdk.api.values.Value value;
```

Comment:
I think that this adapter should be part of the mule implementation instead of something this API provides.

Reply:
This is used by tooling too

Reply:
Tooling should not have to use this class. Currently, tooling uses the `org.mule.runtime.api.value.Value` class, defined in `mule-api`. That shouldn't change. This class should be internal to the SDk implementation

Reply:
yes, this doesn't belong here